src/index.ts

```diff
@@ -168,10 +190,12 @@ export function form<Decl extends FormDeclaration<any>>(
             })
             .remember();
 
+        const _submission$ = DOM.select('form').events('submit', { preventDefault: true });
+        const submission$ = customSubmission$ ? Stream.merge(_submission$, customSubmission$) : _submission$;
```

Comment:
`customSubmission$` is now typed to be `Stream<KeyboardEvent>`, so it is never falsy.

So it should be written like this:
```suggestion
        const submission$ = Stream.merge(_submission$, customSubmission$);
```

(I prefer inlining `_submission$` then.)

Reply:
Fixed in https://github.com/herp-inc/cyclic-form/pull/2/commits/0975fdbc22bcf669670e568efdccb85ff16bbdb9